tracer/resources/applications.py

```diff
@@ -116,7 +116,10 @@ def _append_application(default_attrs, specific_attrs={}):
 	@staticmethod
 	def _helper(app):
 		if app.type == Applications.TYPES["DAEMON"]:
-			return "service {0} restart".format(app.name)
+			if System.init_system() == "systemd":
```

Comment:
I had it this way initially, but I was informed that `service` command is universal across the init systems and there is no need for using `service` there, `systemctl` there and `/etc/init.d/...` elsewhere. Do you have any issues with using `service` on Arch?